tests/blueprint-test.coffee

```diff
@@ -91,14 +121,59 @@ describe "Parsing", ->
         done null
     it "I get the right AST", ->
       assert.deepEqual result.ast,
-        _version: '2.0'
+        _version: '3.0'
         metadata: []
+        element: 'category'
         name: 'API'
         description: ''
+        content: [
+          element: 'category'
+          content: [
+            element: 'resource'
+            name: ''
+            description: ''
+            uriTemplate: '/message'
+            parameters: []
+            model: {}
+            actions: [
+              name: ''
+              description: ''
+              method: 'GET'
+              parameters: []
+              attributes:
+                relation: ''
+                uriTemplate: ''
```

Comment:
Should the lack of an action defined uriTemplate be an empty string instead of null?

IMO, an empty string would be an empty uri and not the lack of a uri.